packages/examples/src/scripts/smart-coffee-machine.ts

```diff
@@ -396,7 +396,7 @@ Assumes one medium americano if not specified, but time and mode are mandatory f
             const paramsp = (await params.value()) as Record<string, unknown>; //  : any = await Helpers.parseInteractionOutput(params);
 
             // Check if uriVariables are provided
-            if (paramsp && typeof paramsp === "object" && "time" in paramsp && "mode" in paramsp) {
+            if (paramsp != null && typeof paramsp === "object" && "time" in paramsp && "mode" in paramsp) {
```

Comment:
Note: What is also "interesting" here is that we cannot simply remove the 
`paramsp != null`
check.

the following code 
```
const paramsp = null;
if (typeof paramsp === "object") {
    console.log("null is object")
}
```

prints `null is object`